test/autoproj/daemon/test_overrides_retriever.rb

```diff
@@ -116,143 +99,117 @@ def create_pull_request(owner, name, number, body, state: "open")
                     )
                 end
             end
-            describe "#retrieve_dependencies" do
-                it "recursively fetches pull request dependencies" do
```

Comment:
Could you keep this test case, please?

Reply:
Same answer. There is no recursive fetching since the dependency refactoring fetches flat and then PullRequest#recursive_dependencies resolve the recursion.